contributors/devel/release/patch-release-manager.md

```diff
@@ -1,258 +1 @@
-script ([example usage](https://gist.github.com/enisoc/058bf0feddf6bffd8e25aa72f9dc38d6)).
-
-## Release cut
-
-A few days before you plan to cut a patch release, put a temporary freeze on
-cherrypick requests by removing the `cherrypick-approved` label from any PR that
-isn't ready to merge.
-Leave a comment explaining that a freeze is in effect until after the release.
-
-The freeze serves several purposes:
-
-1.  It ensures a minimum time period during which problems with the accepted
-    patches may be discovered by people testing on `master`, or by continuous
-    test jobs on the release branch.
-
-1.  It allows the continuous jobs to catch up with `HEAD` on the release branch.
-    Note that you cannot cut a patch release from any point other than `HEAD`
-    on the release branch; for example, you can't cut at the last green build.
-
-1.  It allows slow test jobs like "serial", which has a period of many hours,
-    to run several times at `HEAD` to ensure they pass consistently.
-
-On the day before the planned release, run a mock build with `anago` to make
-sure the tooling is ready.
-If the mock goes well and the tests are healthy, run the real cut the next day.
-
-After the release cut, reapply the `cherrypick-approved` label to any PRs that
-had it before the freeze, and go through the backlog of new cherrypicks.
-
-### Hotfix release
-
-A normal patch release rolls up everything that merged into the release branch
-since the last patch release.
-Sometimes it's necessary to cut an emergency hotfix release that contains only
-one specific change relative to the last past release.
-For example, we may need to fix a severe bug quickly without taking on the added
-risk of allowing other changes in.
-
-In this case, you would create a new, three-part branch of the form
-`release-X.Y.Z`, which [branches from a tag](https://github.com/kubernetes/release/blob/master/docs/branching.md#branching-from-a-tag)
-called `vX.Y.Z`.
-You would then use the normal cherrypick PR flow, except that you target PRs at
-the `release-X.Y.Z` branch instead of `release-X.Y`.
-This lets you exclude the rest of the changes that already went into
-`release-X.Y` since the `vX.Y.Z` tag was cut.
-
-Make sure you communicate clearly in your release plan announcement that some
-changes on the release branch will be excluded, and will have to wait until the
-next patch release.
-
-### Security release
-
-The Product Security Team (PST) will contact you if a security release is needed
-on your branch.
-In contrast to a normal release, you should not make any public announcements
-or push tags or release artifacts to public repositories until the PST tells you to.
-
-See the [Security Release Process](../security-release-process.md) doc for more
-details.
-
+This file has been migrated to https://github.com/kubernetes/sig-release/blob/master/release-process-documentation/release-team-guides/patch-release-manager-playbook.md
```

Comment:
https://github.com/kubernetes/sig-release/blob/master/release-process-documentation/release-team-guides/patch-release-manager-playbook.md is 404

Reply:
now that kubernetes/sig-release#48 has merged it's not